kubernetes/createYaml.sh

```diff
@@ -28,32 +28,36 @@ if [[ "$OSTYPE" == "linux-gnu" ]]; then
   
   echo "Create yaml files done"
   
-  sleep 40s
-  
+  numberOfRnningPods="foo"
+  numberOfAllPods="bar"
+
+  while [ "$numberOfRnningPods" != "$numberOfAllPods" ];
+  do
+    sleep 10s
```

Comment:
@yanmo96 Why we still need to wait 10s here?

Reply:
It checks the status every 10s